hermes/src/api/types.rs

```diff
@@ -78,30 +78,29 @@ impl RpcPriceFeed {
         verbose: bool,
         binary: bool,
     ) -> Self {
-        let price_feed_message = price_feed_update.price_feed;
+        let price_feed = price_feed_update.price_feed;
```

Comment:
```suggestion
        let price_feed = price_feed_update.price_feed;
        let price = price_feed.get_price_unchecked();
```

Nitpick, probably optimises away anyway as you have it.

Reply:
Thanks